.github/dependabot.yml

```diff
@@ -15,25 +15,29 @@ updates:
     ignore:
       - dependency-name: react
         versions:
-          - '^16.11.0'
+          - '>=18.0.0'
       - dependency-name: react-dom
         versions:
-          - '^16.11.0'
+          - '>=18.0.0'
+      - dependency-name: react-test-renderer
+        versions:
+          - '>=18.0.0'
       - dependency-name: '@types/react'
         versions:
-          - '^16.10.0'
+          - '>=18.0.0'
       - dependency-name: '@types/react-dom'
         versions:
-          - '^16.10.0'
+          - '>=18.0.0'
       - dependency-name: '@storybook/react'
         versions:
-          - '^6.0.21'
+          - '^6.1.5'
       - dependency-name: '@storybook/addon-docs'
         versions:
-          - '^6.0.21'
+          - '^6.1.5'
       - dependency-name: '@storybook/addon-viewport'
         versions:
-          - '^6.0.21'
+          - '^6.1.5'
```

Comment:
While I'm at it, I bumped these versions to what we're currently using (since that's what they were before).